cmd/project/ci.go

```diff
@@ -108,10 +108,12 @@ var projectCI = &cobra.Command{
 			}
 		}
 
-		logging.FromContext(cmd.Context()).Infof("Remove unnecessary fonts from tcpdf")
-
-		if err := cleanupTcpdf(args[0]); err != nil {
-			return err
+		tcpdfPath := path.Join(args[0], "vendor", "tecnickcom/tcpdf/fonts")
```

Comment:
can you do the check inside cleanTcpdf method?

Reply:
I did that first, but thought it might be misleading to output "Cleaning tcpdf" anyway. So I moved it out of the method.
Should simply move the output too or just output the message even if the folder might not exist?

Reply:
I would move the output then :)